src/clj/puppetlabs/services/request_handler/request_handler_service.clj

```diff
@@ -2,14 +2,8 @@
   (:require [puppetlabs.trapperkeeper.core :as tk]
             [puppetlabs.services.protocols.request-handler :as handler]
             [puppetlabs.services.request-handler.request-handler-core :as core]
-            [puppetlabs.services.jruby.jruby-puppet-service :as jruby]
             [puppetlabs.trapperkeeper.services :as tk-services]))
 
-(defn- handle-request
-  [request jruby-service config]
-  (jruby/with-jruby-puppet jruby-puppet jruby-service
-    (core/handle-request request jruby-puppet config)))
-
```

Comment:
I guess it's a convention/style thing, but I find it a bit odd to see service references being passed around into namespaces that otherwise have nothing to do with Trapperkeeper.  (e.g. `request-handler-core`).  Makes it a bit more onerous to test, etc.